include/modem/lte_lc.h

```diff
@@ -672,186 +883,240 @@ struct lte_lc_conn_eval_params {
 	uint32_t cell_id;
 };
 
-/** @brief Specifies which type of search the modem should perform when a neighbor
- *	   cell measurement is started.
+/**
+ * Specifies which type of search the modem should perform when a neighbor cell measurement is
+ * started.
  *
- *         When using search types up to LTE_LC_NEIGHBOR_SEARCH_TYPE_EXTENDED_COMPLETE,
- *         result contains parameters from current/serving cell
- *         and optionally up to CONFIG_LTE_NEIGHBOR_CELLS_MAX neighbor cells.
+ * When using search types up to @ref LTE_LC_NEIGHBOR_SEARCH_TYPE_EXTENDED_COMPLETE, the result
+ * contains parameters from current/serving cell and optionally up to
+ * @kconfig{CONFIG_LTE_NEIGHBOR_CELLS_MAX} neighbor cells.
  *
- *         GCI (Global Cell ID) search types are supported with modem firmware versions >= 1.3.4.
- *         Result notification for GCI search types include Cell ID, PLMN and
- *         TAC for up to gci_count (@ref lte_lc_ncellmeas_params) surrounding cells and
- *         optionally, similarly to search types up to
- *         LTE_LC_NEIGHBOR_SEARCH_TYPE_EXTENDED_COMPLETE, a list of neighbor cell measurement
- *         results related to the current_cell (@ref lte_lc_cells_info).
+ * Result notification for Global Cell ID (GCI) search types include Cell ID, PLMN and TAC for up to
+ * @c gci_count (@ref lte_lc_ncellmeas_params) surrounding cells and optionally, similarly to search
+ * types up to @ref LTE_LC_NEIGHBOR_SEARCH_TYPE_EXTENDED_COMPLETE, a list of neighbor cell
+ * measurement results related to the current_cell (@ref lte_lc_cells_info).
```

Comment:
This is not supported currently. Not sure what we should do. I think it could be added once the support is there in the modem, if ever.

```suggestion
 * Result notification for Global Cell ID (GCI) search types include Cell ID, PLMN and TAC for up to
 * @c gci_count (@ref lte_lc_ncellmeas_params) surrounding cells.
```

Reply:
Yes, I agree that this should be removed because currently the modem does not work like that. I'll remove the latter part, let's update the API if modem functionality changes.